src/main/scala/com/fortysevendeg/android/sqlite/resultset/SQLDroidResultSet.scala

```diff
@@ -9,6 +9,7 @@ import java.util.Calendar
 
 import android.database.Cursor
 import com.fortysevendeg.android.sqlite._
+import com.fortysevendeg.android.sqlite.data.{SQLDroidBlob, SQLDroidClob}
```

Comment:
In this PR I've moved `SQLDroidBlob` and `SQLDroidClob` to a subpackage. This is why we need to add the import now (previously they are in the same package)